cupyx/fallback_mode/fallback.py

```diff
@@ -1,14 +1,17 @@
 """
-Main fallback class.
+`fallback_mode` for cupy. Whenever a method is not yet implemented in CuPy,
+it will fallback to corresponding NumPy method.
 """
-
+import sys
 import types
+
 import numpy as np
 
 import cupy as cp
 
-
-from cupyx.fallback_mode import utils
+# -----------------------------------------------------------------------------
+# _RecursiveAttr
+# -----------------------------------------------------------------------------
```

Comment:
This comment seems not needed.

Reply:
You mean blocks of following types?
```python
# -----------------
# block_name
# -----------------
```

Reply:
This comment says the same thing with the class name. I think it is better to remove this redundant information.